README.md

```diff
@@ -39,9 +39,11 @@ well. For more, see the [Code of Conduct](https://swift.org/community/#code-of-c
 
 ## Getting Started
 
-These instructions give the most direct path to a working Swift
-development environment. Options for doing things differently are
-discussed below.
+These instructions give the most direct path to a working Swiftdevelopment
+environment. To build from source you will need 2gb of disk space for the
+source code and over 20gb of disk space for the build artifacts. The first
+build will take multiple hours, but incremental builds will finish much faster.
```

Comment:
So I have a Mid-2015 MBP 2.5 GHz i7, and running `-x` took 3 hours, and just a debug compiler (`-R --debug-swift`) took 70 minutes. So I think this is pretty accurate. My intent with adding this information was to set expectations, not an official benchmark.